package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-ibmdb",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
Do not increment the version, our tooling will do this automatically when we do a release.